package.json

```diff
@@ -62,6 +63,7 @@
     "@openedx-plugins/course-app-teams": "file:plugins/course-apps/teams",
     "@openedx-plugins/course-app-wiki": "file:plugins/course-apps/wiki",
     "@openedx-plugins/course-app-xpert_unit_summary": "file:plugins/course-apps/xpert_unit_summary",
+    "@openedx/frontend-build": "^14.0.14",
```

Comment:
`frontend-build` is required for webpack to build the MFE so it should be a dependency, not a devDependency.

Ref: [OEP-67 - Frontend Technology Selection - 15 Development dependencies should be separated into devDependencies](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0067-bp-tools-and-technology.html#frontend-technology-selection)